namespaces/Representor.md

```diff
@@ -0,0 +1,141 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Object Element)
+
+The Resource represents a Hypermedia Resource, with it's available relations, transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `content`: (object)
+    - `relations` - (array[Relation]) - The relations from this resource.
```

Comment:
What is the purpose of `relations` if you have `transitions`? I believe `transition` is listing all the possible transition including their relations?